glados-web/assets/js/radiusdensity.js

```diff
@@ -76,3 +86,25 @@ function radius_node_id_scatter_chart(data) {
         .on("mouseout", hoverGone);
 
 }
+
+function getClientStringFromDecodedEnr(enr) {
+    for (let [key, value] of ENR.ENR.decodeTxt(enr).enr.entries()) {
+
+        if (key === "c") {
+            let fullClientString = String.fromCharCode.apply(null, value);
+            if (fullClientString[0] === 'f') {
+                return "fluffy";
+            } 
+            else if (fullClientString[0] === 't') {
+                clientName = "trin ";
+                clientName += fullClientString.substring(2);
+                return clientName;
+            } else {
```

Comment:
```suggestion
            else if  (fullClientString[0] === 'u')
```
include ultralight too